msgbox/msgbox.js

```diff
@@ -42,9 +42,11 @@ function messageBox({
   }
 
   function resolveWith(value) {
-    setTimeout(messageBox.resolve, 0, value);
-    animateElement(messageBox.element, {className: 'fadeout', remove: true})
-      .then(unbindAndRemoveSelf);
+    if (messageBox.resolve) {
```

Comment:
Without this check, I was getting errors due to `messageBox.resolve` being `null` - this was occurring after I hit escape to close the help-popup, but only after `messageBox` was previously visible and closed.

Reply:
It means the keydown listener was never detached so a better solution is to investigate why. Turns out there was a typo: `document.removeEventListener('keydown'` should be `window.`. Also, `Esc` key should only close messageBox when both are displayed. I've push a proper fix in master so please revert this change.